server/client_handler.go

```diff
@@ -298,3 +292,20 @@ func parseLine(line string) (string, string) {
 	}
 	return params[0], params[1]
 }
+
+// logError logs an error if err is not an error we would expect for closed connections.
+func logError(l *logrus.Entry, msg string, err error) {
+	if closedConnErr(err) {
+		return
```

Comment:
do we want to ignore these entirely? or would it be better to only show these errors at debug/trace level? given there are largely useless, I guess not, but raising the question.